frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/cell/BranchOrCommitCellRendererComponent.java

```diff
@@ -242,6 +250,19 @@ private static String getLabel(SyncToRemoteStatus status) {
         Case($(DivergedFromAndOlderThanRemote), "  (diverged from & older than ${remoteName})"));
   }
 
+  private static String getSyncToParentStatusBasedTooltip(IGitMacheteNonRootBranch currentBranch) {
+    var currentBranchName = currentBranch.getName();
+    var parentBranchName = currentBranch.getParentBranch().getName();
+    return Match(currentBranch.getSyncToParentStatus()).of(
+        Case($(InSync), "Branch '${currentBranchName}' is in sync with '${parentBranchName}'"),
```

Comment:
Pls extract to bundles

Reply:
Not that we care about i18n specifically... but better to keep user-facing messages close to each other, it makes spotting mistakes easier (although I do agree that a bit at the expense of the benefits provided by string interpolation :frowning_face:)